support/cas-server-support-radius/src/main/java/org/apereo/cas/config/RadiusConfiguration.java

```diff
@@ -42,6 +43,7 @@
 @Slf4j
 public class RadiusConfiguration {
 
+    private static final Splitter IPS = Splitter.on(" ").trimResults().omitEmptyStrings();
```

Comment:
Please use `,` instead to be consistent with other properties, and use `org.springframework.util.StringUtils.commaDelimitedListToSet` to parse the values. Please update the documentation for the field to explain the new behavior with examples.